backup_and_restore/application_backup_and_restore/installing/uninstalling-oadp.adoc

```diff
@@ -6,4 +6,19 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You uninstall the OpenShift API for Data Protection (OADP) by deleting the OADP Operator. See xref:../../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-cluster[Deleting Operators from a cluster] for details.
+You uninstall the OpenShift API for Data Protection (OADP) by deleting the OADP Operator.
+
+[id="uninstalling-oadp-web-console"]
+== Deleting Operators from a cluster using the web console
+
+Cluster administrators can delete installed Operators from a selected namespace by using the web console. See xref:../../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-web-console[Deleting Operators from a cluster using the web console].
+
+[id="uninstalling-oadp-cli"]
+== Deleting Operators from a cluster using the CLI
+
+Cluster administrators can delete installed Operators from a selected namespace by using the CLI. See xref:../../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-cli[Deleting Operators from a cluster using the CLI].
+
+[IMPORTANT]
+====
+You must successfully and completely uninstall the OADP Operator prior to attempting to reinstall.
+====
```

Comment:
Does this need to be in two headings? 


[id="uninstalling-oadp-web-console"]
== Deleting Operators from a cluster
Cluster administrators can delete installed Operators from a selected namespace by using xref:../../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-web-console[the web console] or  xref:../../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-cli[the CLI].

Reply:
cool, done